doc/source/theory/dem/dem.rst

```diff
@@ -160,6 +160,67 @@ The tangential overlap is calculated with the tangential force with no damping f
 
 Regarding the particle-wall contacts, applied models are the same than particle-particle contacts with a background triangulation and mapping with walls.
 
+---------------------------------------------
+Johnson-Kendall-Roberts force model
+---------------------------------------------
+The Johnson-Kendall-Roberts (JKR) force model enable the modelisation of attractive forces due to the van der Waals effects.
+This model modifies the Hertz formulation by defining a larger contact path radius (:math:`\mathbf{a}`) and by taking into account a new parameter, the effective surface energy (:math:`\mathbf{\gamma}_{e}`).
+The model is define by the following equation.
+
+.. math::
+    a^{3} = \frac{3 R_{e}}{4 E_{e}} \left[F_{n}^{JKR} + 3\pi\gamma_{e}R_{e}  + \sqrt{6 F_{n}^{JKR} \pi\gamma_{e}R_{e} + (3\pi\gamma_{e}R_{e})^2 }\right]
+
+Where :math:`\mathbf{F_{n}^{JKR}}` correspond to the normal spring force and attractive force combine and :math:`\mathbf{\gamma_{e}}` is the effective surface energy.
+Note that if the effective surface energy is equal to zero, the JKR model goes back the Hertz definition.
+
+The effective surface energy can be computed like so :
+
+.. math::
+    \gamma_{e} = \gamma_{1} + \gamma_{2} - 2\gamma_{1,2}
+
+Where :math:`\gamma_{1}` and :math:`\gamma_{2}` are the surface energy of each material (particle or wall) :math:`\gamma_{1,2}` is the interface energy which is equal to zero when both surface are coming form the same material.
+In Lethe, the interface energy term is approximated using : `[3] <https://doi.org/10.1016/B978-0-12-391927-4.10013-1>`_
+
+.. math::
+    \gamma_{1,2} \approx \left( \sqrt{\gamma_{1}} - \sqrt{\gamma_{2}}  \right)^{2}
+
+To compute the :math:`\mathbf{F_{n}^{JKR}}`, the contact patch radius need to be determied. The contact patch radius can be related to the normal overlap as follow:
+
+.. math::
+    \delta_{n} = \frac{ a^{2} }{ R_{e} } -  \sqrt{ \frac{2 \pi \gamma_{e} a }{ Y_{e} } }
+
+This equation can be rewrited as a fourth order polynomial function with two imaginary root and two real root.
+
+.. math::
+    0 = a^{4} - 2R_{e}\delta_{n}a^{2} - 2\pi\gamma_{e}R_{e}^{2}a + R_{e}^{2}\delta_{n}^{2}
+
+Since we are always trying to solve for the same real root, a straightforward procedure, describe by Parteli et al. can be use. `[4] <https://doi.org/10.1038/srep06227>`_
```

Comment:
```suggestion
Since we are always trying to solve for the same real root, a straightforward procedure, describe by Parteli et al. can be used `[4] <https://doi.org/10.1038/srep06227>`_ :
```